Makefile

```diff
@@ -0,0 +1,21 @@
+SNAPPY = $(CHAIN)/vendor/github.com/google/snappy
+ROCKSDB = $(CHAIN)/vendor/github.com/facebook/rocksdb
+
+null:
+	@echo Please specify a make target.
+
+## builds chain core with c dependencies, for development environments
+build-dev: build-lib
+	go install -tags 'localhost_auth http_ok reset' chain/cmd/cored
```

Comment:
Just in case the user has the `chain` repo checked out under some name other than `chain`, it might be better to do
```
cd $(CHAIN) && go install -tags 'localhost_auth http_ok reset' ./cmd/cored
```